CleverPush/CleverPush/Source/CleverPush.m

```diff
@@ -392,6 +398,25 @@ + (void)initWithChannelId {
     [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(applicationWillTerminate) name:UIApplicationWillTerminateNotification object:nil];
 }
 
++ (void)initialisedTopicDialogData:(NSDictionary*)Config {
+    NSArray* channelTopics = [Config valueForKey:@"channelTopics"];
+    if (channelTopics != nil && [channelTopics count] > 0) {
+        NSArray* topics = [self getSubscriptionTopics];
+        
+        if (!topics || [topics count] == 0) {
+            NSMutableArray* selectedTopicIds = [[NSMutableArray alloc] init];
+            for (id channelTopic in channelTopics) {
+                if (channelTopic != nil && ([channelTopic valueForKey:@"defaultUnchecked"] == nil || ![[channelTopic valueForKey:@"defaultUnchecked"] boolValue])) {
+                    [selectedTopicIds addObject:[channelTopic valueForKey:@"_id"]];
+                }
+            }
+            if ([selectedTopicIds count] > 0) {
+                [self setSubscriptionTopics:selectedTopicIds];
```

Comment:
here we will call `setSubscriptionTopics` which is triggering a sync / subscribe. This is not what we want.